applications/MappingApplication/README.md

```diff
@@ -0,0 +1,182 @@
+## Mapping Application
+The Mapping Application contains the core developments in mapping data between non matching grids. It works both in shared and distributed (**MPI**) memory environments as well as in 1D, 2D and 3D domains.
+
```

Comment:
I would make the feature list more visible:
* Serial/OpenMP/MPI
* 1D/2D/3D
* Matching/NonMatching
* Mappers (Nearest Neighbors, ...)
* Mapping operations: (ADD_VALUES, SWAP_SIGN, etc.)

I would also add a list of supported mesh topologies (shapes, linear/quadratic, ...)

Reply:
done